qiskit/transpiler/target.py

```diff
@@ -539,30 +550,44 @@ def operations_for_qargs(self, qargs):
         Raises:
             KeyError: If qargs is not in target
         """
-        if qargs not in self._qarg_gate_map:
+        if qargs is not None and any(x not in range(0, self.num_qubits) for x in qargs):
             raise KeyError(f"{qargs} not in target.")
         res = [self._gate_name_map[x] for x in self._qarg_gate_map[qargs]]
-        if None in self._qarg_gate_map:
-            res += [self._gate_name_map[x] for x in self._qarg_gate_map[None]]
-        return res
+        if qargs is not None:
+            res += self._global_operations.get(len(qargs), list())
```

Comment:
I actually thought there was a pylint rule that complained about using `list()` or `dict()` in place of the empty literals, but if so, we must have it disabled.

Reply:
I don't see it in the global disable list: https://github.com/Qiskit/qiskit-terra/blob/main/.pylintrc#L61-L76 and the only file specific disable is `too-many-return-statements`. I can change this to `[]` if you'd prefer. I think I just used list because the other method used `set()` and it was in my head.

Reply:
No, I don't care - I just thought pylint had moaned at me before. Maybe it's in a newer version?